app/views/audit_mailer/summary.html.erb

```diff
@@ -11,10 +11,10 @@
            user_search_param = login ? "user = #{login}" : %Q{username = "#{audit.username}"}
         %>
         <b>
-          <%= link_to(audit.username, audits_path(:host => @url.host, :port => @url.port, :only_path => false, :protocol => @url.scheme, :search => user_search_param)) %>
+          <%= link_to(audit.username, audits_url(:host => @url.host, :port => @url.port, :protocol => @url.scheme, :search => user_search_param)) %>
```

Comment:
I think so, because otherwise it'll render the URL based on the current request that it's processing.  The `@url` variable is populated from the `foreman_url` setting which might be different, it's there to allow the user to override the public URL of Foreman used in the mailers etc.

I don't think using the *_url handlers results in anything different than with _path + only_path => false.